### ISSUE TITLE
folder collapse

### ISSUE THREAD

**Issue by laoshaw** [cjsheets/mkdocs-rtd-dropdown#10]
this is a great plugin, just one more addition will make it perfect.

mkdocs honor sub-folders under docs/ and will generate all html pages for markdown files under them without modifying mkdocs.yml to add the pages manually, which is super handy when you have a large set of markdown files in a hierarchy layout.

this plugin can collapse for each page with its heading sections, but not for all the pages under the folder name when you click on the folder name.

say, i have:

```
docs/folder1/file1.md
docs/folder1/file2.md
```
if I can click on "folder1" it will collapse that will be great.

**Comment by cjsheets** [comment]
I agree that collapsing based on nested folders would be ideal. It's a bit difficult to implement though.

If you look at how the official [ReadTheDocs](https://docs.readthedocs.io/en/latest/gsoc.html#project-ideas) parser works, you'll see they don't have a concept of nested pages. They only support very long pages that collapse based on heading-level.

That said, I agree that collapsing based on folder structure is best in the context of mkdocs.

I'll see if it's possible to implement this as a feature that could be turned on or off via configuration option.
